aws_lambda/src/gateway.rs

```diff
@@ -4,7 +4,7 @@ use futures::IntoFuture;
 use aws_lambda_gateway::NewApiGatewayProxy;
 use aws_lambda_runtime::Handler;
 
-pub use aws_lambda_gateway::{http, Body};
+pub use aws_lambda_gateway::{http, Body, PathParameters, QueryParameters};
```

Comment:
I think it might be useful to export these only if the `gateway_extensions` feature is enabled.

Reply:
Added